lib/epoch.ex

```diff
@@ -1,27 +1,14 @@
 defmodule DateTimeParser.Epoch do
   @moduledoc false
 
-  import NimbleParsec
-  import DateTimeParser.Formatters, only: [format: 1]
-  import DateTimeParser.Combinators.Epoch
-
-  defparsec(:parse, unix_epoch())
+  def parse(%{"epoch" => epoch}) do
+    with {epoch_as_float, _} <- Float.parse(epoch),
+         microsecond_epoch <- round(epoch_as_float * :math.pow(10, 6)) do
```

Comment:
I was going with `round` over `trunc` becuase I _think_ `.0000001` (six zeros and then a 1) would make sense to be all zeros, but `.0000009` (six zeros and then a 9) would make sense to be rounded up. I would want to have some tests that show this behavior. 

`:math.pow(10,6)` is converting seconds into microseconds.